docs/runbooks/HPPNotReady.md

```diff
@@ -0,0 +1,31 @@
+# HPPNotReady
+
+## Meaning
+
+The HPPNotReady alert is indicative of a HPP installation being in a degraded state;  
```

Comment:
@akalenyu Please write here what HPP is responsible of.

Reply:
Done